apps/vr-tests-react-components/.storybook/preview.js

```diff
@@ -25,28 +34,25 @@ setAddon({
    * @this {import('../src/utilities/types').ExtendedStoryApi}
    */
   addStory(storyName, storyFn, config = {}) {
+    const modeConfig = { ...configDefaults, ...config };
     this.add(storyName, context => {
-      return <FluentProvider theme={webLightTheme}>{storyFn(context)}</FluentProvider>;
+      return (
+        <>
+          {Object.keys(modeConfig).map(configProp => {
+            const { label, ...providerConfig } = configToPropsMap[configProp];
+            return (
+              <React.Fragment key={configProp}>
+                <h3>{label}</h3>
+                <FluentProvider key={configProp} {...providerConfig}>
```

Comment:
```suggestion
                <FluentProvider {...providerConfig}>
```

`key` there is redundant